objects/ship/avalitechstation/avalitechstation.object

```diff
@@ -35,8 +37,7 @@
 
     }
   ],
-  
-  "turnInQuests" : [ "gaterepair" ],
+
   "offeredQuests" : [ "gaterepair" ],
```

Comment:
I take it this is the change that changes it from instant progression to the little chat SAIL is supposed to give?